packages/components/src/horizontal-bar-list/style.scss

```diff
@@ -67,7 +67,7 @@
 	.horizontal-bar-list--hover-action {
 		display: none;
 		position: relative;
-		z-index: 1;
+		z-index: 2;
```

Comment:
```suggestion
```

Does removing this z-index work as your expected? It could make the `ul.module-content-list-item-actions` work with the z-index: 2 to create the stacking context covering anything behind its z-index.

The original `.horizontal-bar-list--hover-action` would be covered by the next `.horizontal-bar-list--hover-action` due to the HTML structure when setting the z-index.

<img width="970" alt="截圖 2022-11-17 上午11 13 36" src="https://user-images.githubusercontent.com/6869813/202347959-11f9b3fb-2aac-4bca-893f-b67521ec3887.png">

<img width="928" alt="截圖 2022-11-17 上午11 26 01" src="https://user-images.githubusercontent.com/6869813/202347975-2dff0866-2a31-48b0-b01c-d2ac70ce3aa5.png">

Reply:
Yup, this fixed it. Thank you!